src/server/database/heuristics.js

```diff
@@ -8,9 +8,15 @@
 */
 
 // --------- Fake Heuristics ---------------
-function run(layouts, callback) {
+function run(layouts, trimValues, callback) {
     let layout = layouts.toArray()
       .then((result) => {
+
+        //Return last 10 layouts
+        if(trimValues){
```

Comment:
You should just accept a general number param here and return result.slice(num).  It is more flexible that way.

Reply:
All the layout functions now take in numEntries to allow the caller to define the number of layouts returned.